api/api/src/main/scala/uk/ac/wellcome/platform/api/elasticsearch/ImagesQuery.scala

```diff
@@ -3,33 +3,44 @@ package uk.ac.wellcome.platform.api.elasticsearch
 import com.sksamuel.elastic4s.ElasticDsl._
 import com.sksamuel.elastic4s.Index
 import com.sksamuel.elastic4s.requests.common.DocumentRef
-import com.sksamuel.elastic4s.requests.common.Operator.AND
+import com.sksamuel.elastic4s.requests.common.Operator.{AND, OR}
 import com.sksamuel.elastic4s.requests.searches.queries.{BoolQuery, Query}
-import com.sksamuel.elastic4s.requests.searches.queries.matches.MultiMatchQueryBuilderType.CROSS_FIELDS
+import com.sksamuel.elastic4s.requests.searches.queries.matches.MultiMatchQueryBuilderType.{
+  BEST_FIELDS,
+  CROSS_FIELDS
+}
 import com.sksamuel.elastic4s.requests.searches.queries.matches.{
   FieldWithOptionalBoost,
   MultiMatchQuery
 }
+import uk.ac.wellcome.elasticsearch.WorksAnalysis.whitespaceAnalyzer
 
 case object ImagesMultiMatcher {
   def apply(q: String): BoolQuery = {
     val fields = Seq(
-      "data.subjects.concepts.label",
-      "data.genres.concepts.label",
-      "data.contributors.agent.label",
-      "data.title.english",
-      "data.description.english",
-      "data.alternativeTitles.english",
-      "data.physicalDescription.english",
-      "data.production.*.label",
-      "data.language.label",
-      "data.edition",
-      "data.notes.content.english",
-      "data.collectionPath.path",
-      "data.collectionPath.label",
-    ).flatMap { field =>
-      Seq(s"source.canonicalWork.$field", s"source.redirectedWork.$field")
-    } map (FieldWithOptionalBoost(_, None))
+      ("data.contributors.agent.label", Some(1000)),
+      ("data.title", Some(100)),
+      ("data.title.english", Some(100)),
+      ("data.title.shingles", Some(100)),
+      ("data.alternativeTitles", Some(100)),
+      ("data.subjects.concepts.label", Some(10)),
+      ("data.genres.concepts.label", Some(10)),
+      ("data.production.*.label", Some(10)),
+      ("data.description", None),
+      ("data.physicalDescription", None),
+      ("data.language.label", None),
+      ("data.edition", None),
+      ("data.notes.content", None),
+      ("data.lettering", None),
+      ("data.collectionPath.path", None),
+      ("data.collectionPath.label", None),
```

Comment:
Tiny nit: the horizontal alignment of these makes it quite tricky to see which fields have the same boost. Possibly flip them round to make it easier to see the relative boost on the different fields?

e.g.

```scala
val boostedFields = Map(
      Some(1000) -> Seq(
        "data.contributors.agent.label"
      ),
      Some(100) -> Seq(
        "data.title",
        "data.title.english",
        "data.title.shingles",
        "data.alternativeTitles",
      ),
      Some(10) -> Seq(
        "data.subjects.concepts.label",
        "data.genres.concepts.label",
        "data.production.*.label",
      ),
      None -> Seq(
        "data.description",
        "data.physicalDescription",
        "data.language.label",
        "data.edition",
        "data.notes.content",
        "data.lettering",
        "data.collectionPath.path",
        "data.collectionPath.label"
      )
    )

    val fields: Map[String, Option[Int]] =
      boostedFields
        .flatMap { case (boost, fieldNames) =>
          fieldNames.map { _ -> boost }
        }
        .flatMap { case (fieldName, boost) =>
          toWorkField(fieldName).map { workField => (workField, boost) }
        }
        .map { case (field, boost) =>
          FieldWithOptionalBoost(field, boost.map(_.toDouble))
        }
```

Reply:
The point it to not couple things in this regard, but it does seem good to understand the order. Thanks!